docs/csharp/language-reference/tokens/interpolated.md

```diff
@@ -94,6 +94,20 @@ The following example uses implicit conversion to <xref:System.FormattableString
 
 :::code language="csharp" source="./snippets/string-interpolation.cs" id="Snippet4":::
 
+## Combining multiple interpolated strings
+
+To combine multiple interpolated strings, use the interpolation expression for each:
+
+```csharp
```

Comment:
ℹ️ I'm happy to move this to a sample code file if desired; just seemed like overkill in this instance.

Reply:
This is fine as is, I agree it's rather simple.